src/components/dashboard/FaceRecognition/FaceRecognition.js

```diff
@@ -62,85 +58,51 @@ class FaceRecognition extends React.Component<FaceRecognitionProps, State> {
 
   height = 0
 
-  componentWillUnmount = () => {
-    log.debug('Unloading ZoomSDK', this.loadedZoom)
-    this.timeout && clearTimeout(this.timeout)
-  }
-
-  componentWillMount = async () => {
-    await zoomSdkLoader.ready
-    this.loadedZoom = ZoomSDK
-
-    navigator.getMedia =
-      navigator.getUserMedia || // use the proper vendor prefix
-      navigator.webkitGetUserMedia ||
-      navigator.mozGetUserMedia ||
-      navigator.msGetUserMedia
-
-    navigator.getMedia(
-      { video: true },
-      () =>
-        (this.timeout = setTimeout(() => {
-          this.setState({ zoomReady: true })
-        }, 0)),
-      this.showFRError
+  uuidv4() {
```

Comment:
there are ready-to-use uuid libraries - you could use one of them

Reply:
`import { v4 as uuidv4 } from 'uuid'`

Reply:
done in https://github.com/GoodDollar/GoodDAPP/pull/1341/commits/44a0fdf04a8750dde4707827f97a7d4732e4e649